inc/saiqueue.h

```diff
@@ -70,6 +70,9 @@ typedef enum _sai_queue_attr_t
     /** Attach scheduler to Queue [sai_object_id_t]*/
     SAI_QUEUE_ATTR_SCHEDULER_PROFILE_ID = 0x00000003,
 
+    /** PFC based pause state [booldata] */
```

Comment:
should not restricted to pfc, just queue pause state.
default false. (true: means queue in pause state).